libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -445,6 +445,7 @@ dictionary ReverseSwapFeesRequest {
 dictionary OnchainPaymentLimitsResponse {
     u64 min_sat;
     u64 max_sat;
+    u64 max_sat_with_current_channels;
```

Comment:
IMO we don't need it. User can take it from the node info.

Reply:
Are you sure?

Sending onchain is already 3 steps:
- `onchain_payment_limits`
- `prepare_onchain_payment`
- `pay_onchain`

Keeping this out of the `onchain_payment_limits` result means sending onchain will need 4 steps.

Reply:
It seems weird to return mixture of onchain limits with user balance to me. Application developers usually already cache the user balance somewhere (and update it whenever they get a change event from the sdk) so I am not sure they will need to call it explicitly during the call.
IMO it is cleaner without it but happy to hear more opinions regarding this.

Reply:
The solution discussed (app checking `max_reverse_swap_amount` separately) was already in place for a while, but was removed with #850 to simplify the steps.

How about this as an alternative solution: `onchain_payment_limits` doesn't return the `max_reverse_swap_amount`, but throws an error if that value is below the swapper minimum? This way, instead of returning limits `min = 0, max = 0` as it does now, it would result in an error. Would that work?

Reply:
I don't think so because the user wants to know on any case the swapper original limits. That was the issue from the first place.

Reply:
Alright then I will add the separate method.